src/index.css

```diff
@@ -96,3 +96,9 @@ button:hover:not(.orange) {
   display: grid;
   gap: 50px;
 }
+
```

Comment:
[ **OPTIONAL**] 
-  Great job with styling, but I propose you add more styling to improve the feel of the website.
![image](https://github.com/tamoorsaeed22/Math-Magician/assets/62849872/5cb72dbb-5d8e-45ce-8d8e-72fd2b6a8dd8)